pvm/ts/src/engine/texasHoldem-ante-headsup.test.ts

```
@@ -0,0 +1,67 @@
+import { TexasHoldemRound, GameOptions } from "@bitcoinbrisbane/block52";
+import TexasHoldemGame from "./texasHoldem";
+
+import { ethers } from "ethers";
+import { gameOptions } from "./testConstants";
+
+// This test suite is for the Texas Holdem game engine, specifically for the Ante round in a heads-up scenario.
+describe.only("Texas Holdem - Ante - Heads Up", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)